assets/src/apps/authoring/components/AdaptivityEditor/ActionFeedbackEditor.tsx

```diff
@@ -92,6 +92,7 @@ const ActionFeedbackEditor: React.FC<ActionFeedbackEditorProps> = ({
           id={`action-feedback-${uuid}`}
           placeholder="Enter feedback"
           disabled={false}
+          readOnly = {true}
```

Comment:
Making the text box read only so that user can't edit it without opening the text editor.